packages/xo-web/src/xo-app/host/tab-general.js

```diff
@@ -227,6 +241,27 @@ export default decorate([
             </Col>
           </Row>
         )}
+        <br />
```

Comment:
This shouldn't be necessary.

Reply:
If I delete it, all the info is on the same line 
![image](https://github.com/user-attachments/assets/5da5d5f5-1ee0-4103-b430-6b0e1a3f7161)